test/openshift/e2e/sequential/1-077_validate_workload_status_monitoring_alert/02-verify-alert.yaml

```diff
@@ -1,13 +1,13 @@
-# apiVersion: kuttl.dev/v1beta1
-# kind: TestStep
-# commands:
-# - script: sleep 10m
-# - script: |
-#     jq '.data.alerts' <<< "$(oc -n openshift-monitoring exec -c prometheus prometheus-k8s-0 -- curl -s   'http://localhost:9090/api/v1/alerts')" > alerts.json
+apiVersion: kuttl.dev/v1beta1
+kind: TestStep
+commands:
+- script: sleep 10m
+- script: |
+    jq '.data.alerts' <<< "$(oc -n openshift-monitoring exec -c prometheus prometheus-k8s-0 -- curl -s   'http://localhost:9090/api/v1/alerts')" > alerts.json
 
-#     result=$(jq --arg deployment "openshift-gitops-applicationset-controller" --arg namespace $NAMESPACE '.[] | select(.labels.alertname == "ApplicationSetControllerNotReady" and .state == "firing")' alerts.json)
+    result=$(jq --arg deployment "openshift-gitops-applicationset-controller" --arg namespace "openshift-gitops" '.[] | select(.labels.alertname == "ApplicationSetControllerNotReady" and .state == "firing")' alerts.json)
```

Comment:
Why do we need `--arg deployment` and `--arg namespace` here? It would define `deployment` and `namespace` variables for jq but no filtering would happen.  I'm wondering if they should rather go to `select` as `.labels.deployment` and `.labels.namespace`